src/info/index.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layouts/base.njk
+title: Info
+templateClass: tmpl-post
+---
+
+<h1 class="mv5 mw5 center fill-accent fg-accent animation-logo animate-me">
+  {% include "svg/weeklies_logo.svg" %}
+</h1>
+
+<article class="pt4 pt5-ns pb4 vs4" contentEditable>
+  {% Heading level=1 %}
+    This week on Tech Weeklies
+  {% endHeading %}
+
+  <ol>
```

Comment:
Would it be better to have only this block be `contenteditable`? Or should the title be editable as well?

Reply:
My idea was to have the title as well editable. Use case: Last friday we started at 15:30 instead of the usual 15:00. I opened the site and edited the dom to display that today we start at 15:30.